presto-docs/src/main/sphinx/release/release-0.211.rst

```diff
@@ -0,0 +1,51 @@
+=============
+Release 0.211
+=============
+
+General Changes
+---------------
+
+* Fix missing final query plan in ``QueryCompletedEvent``. Statistics and cost estimates
+  are removed from the plan text because they may not be available during event generation.
+* Update the default value of the ``http-server.https.excluded-cipher`` config
+  property to exclude non FS cipher suites. Specifically, all
```

Comment:
Let's change this to "forward secrecy (FS)", then we can use the FS acronym in the next sentence. (it's strange to use an acronym first and then define it later)